tck/src/test/java/org/eclipse/microprofile/jwt/tck/config/PublicKeyEndpoint.java

```diff
@@ -32,6 +32,7 @@
 import java.security.interfaces.RSAPublicKey;
 import java.util.Base64;
 import java.util.Optional;
+import java.util.Set;
```

Comment:
@brutif As suggested at #174 please also create a `jaxrs/AudienceValidationEndpoint` as there is a pattern there in `jaxrs` that every test has a dedicated endpoint. This `AudienceValidationEndpoint` would only have those audiences injected.

Reply:
Oops, you've already do it :-), but please revert the injection code in `config/PublickeyEndpoint` :-) as it is not used anywhere there

Reply:
done